nucypher/cli/main.py

```diff
@@ -140,15 +143,16 @@ def status(click_config, config_file):
 @click.argument('action')
 @click.option('--debug', '-D', help="Enable debugging mode", is_flag=True)
 @click.option('--dev', '-d', help="Enable development mode", is_flag=True)
-@click.option('--dry-run', help="Execute normally without actually starting the node", is_flag=True)
+@click.option('--quiet', '-Q', help="Disable logging", is_flag=True)
+@click.option('--dry-run', '-x', help="Execute normally without actually starting the node", is_flag=True)
 @click.option('--force', '-f', help="Don't ask for confirmation", is_flag=True)
 @click.option('--teacher-uri', help="An Ursula URI to start learning from (seednode)", type=click.STRING)
 @click.option('--min-stake', help="The minimum stake the teacher must have to be a teacher", type=click.INT, default=0)
 @click.option('--rest-host', help="The host IP address to run Ursula network services on", type=click.STRING)
 @click.option('--rest-port', help="The host port to run Ursula network services on", type=NETWORK_PORT)
 @click.option('--db-filepath', help="The database filepath to connect to", type=click.STRING)
 @click.option('--checksum-address', help="Run with a specified account", type=EIP55_CHECKSUM_ADDRESS)
-@click.option('--federated-only', help="Connect only to federated nodes", is_flag=True, default=FEDERATED_ONLY)
+@click.option('--federated-only', '-F', help="Connect only to federated nodes", is_flag=True, default=FEDERATED_ONLY)
```

Comment:
I'd be careful with shadowing `-f`. Seems kinda spooky.

Reply:
Yeah - Perhaps let's not have `-f` and require the long form `--force`?

Reply:
Yeah, that makes sense. Any objections from anyone else?

Reply:
+1

Reply:
@KPrasch any changes in thought on this?

Reply:
:heavy_check_mark: